pywwt/tests/test_qt_widget.py

```diff
@@ -45,6 +45,11 @@ def test_methods(self, capsys):
         self.widget.center_on_coordinates(M42, fov=10 * u.deg)
         wait_and_check_output(1, capsys)
 
+    def test_coordinates(self, capsys):
+        self.widget.center_on_coordinates(M42, fov=10 * u.deg)
+        M42 == self.widget.get_center()
```

Comment:
You need to add ``assert`` at the start of the line here to make sure the test fails if the value is False. Instead of checking equality (which won't work anymore) you could use the ``separation`` method on coordinates and check the separation is less than a certain threshold - here's an example of how to use it:

```
In [6]: SkyCoord.from_name('M32').separation(SkyCoord.from_name('M42')).arcsec
Out[6]: 291511.6229661299
```